web/modules/custom/cdp_core/src/Controller/CdpController.php

```diff
@@ -0,0 +1,16 @@
+<?php
+
+namespace Drupal\cdp_core\Controller;
+
+use Drupal\Core\Controller\ControllerBase;
+
+
+class CdpController extends ControllerBase {
+
+  public function content() {
```

Comment:
Kai commitiname kodą, stengiamės jį pateikti kolegoms be įvairių test ir debug funkcijų.